### PR TITLE
analytics: send local room state

### PR DESCRIPTION
**Pull request by shishirng**

### CODE REVIEW

File: livekit_analytics.proto [hunk 1]
```diff
@@ -152,3 +153,27 @@ message AnalyticsEvent {
...
+  string id = 1;
+  string name = 2;
+  google.protobuf.Timestamp created_at = 3;
+  uint32 num_participants = 4;
+  repeated AnalyticsRoomParticipant participants = 5;
+}
+
+message AnalyticsNodeRooms {
+  string node_id = 1;
+  uint64 sequence_number = 2;
```
Comment (davidzhao) [inline livekit_analytics.proto:176]: what is this for?

File: livekit_analytics.proto [hunk 2]
```diff
@@ -152,3 +153,27 @@ message AnalyticsEvent {
...
+  ParticipantInfo.State state = 4;
+  ParticipantInfo.Kind kind = 5;
+  google.protobuf.Timestamp joined_at = 6;
+}
+
+message AnalyticsRoom {
+  string id = 1;
+  string name = 2;
+  google.protobuf.Timestamp created_at = 3;
+  uint32 num_participants = 4;
```
Comment (davidzhao) [inline livekit_analytics.proto:170]: what do we use these for?

File: livekit_analytics.proto [hunk 3]
```diff
@@ -152,3 +153,27 @@ message AnalyticsEvent {
 message AnalyticsEvents {
   repeated AnalyticsEvent events = 1;
 }
+
+message AnalyticsRoomParticipant {
+  string id = 1;
+  string identity = 2;
+  string name = 3;
+  ParticipantInfo.State state = 4;
+  ParticipantInfo.Kind kind = 5;
```
Comment (davidzhao) [inline livekit_analytics.proto:162]: nbd, but do we need name and kind? is the idea to reconstruct things from here?

File: livekit_analytics.proto (hunk 1 above)
Comment (shishirng) [reply livekit_analytics.proto:176]: auto incremented counter to stamp messages. Will be useful in debugging/tracking missed messages

File: livekit_analytics.proto (hunk 2 above)
Comment (shishirng) [reply livekit_analytics.proto:170]: will remove, its redundant, we can infer from message

File: livekit_analytics.proto (hunk 3 above)
Comment (shishirng) [reply livekit_analytics.proto:162]: thats right, if there are missed join/active event and participant was still active, we could reconstruct participant with the data

**Review by davidzhao — APPROVED** [review]
lgtm!

**Pull request merged**